ChessBoard.Api/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: "3"
+services:
+  mongo:
```

Comment:
Right now only way I can think of is running the mongo in a container, but running the app itself locally (since everything is in place to build/resolve and run the project locally in seconds), therefore you can stop it, edit the code, run it again, while the mongo container keeps running. Looking forward to your feedback

Reply:
yes, that would be a good approach - provide connection string to docker container but run the app from windows env.